scripts/pr.star

```diff
@@ -50,6 +51,7 @@ def pr_pipelines(edition):
         build_docker_images_step(edition=edition, ver_mode=ver_mode, archs=['amd64',]),
         postgres_integration_tests_step(),
         mysql_integration_tests_step(),
+        release_canary_npm_packages_step(edition),
```

Comment:
Don't forget to revert changes in this file before merging to master :)

Reply:
Thanks for the reminder. Reverted it! 👍